### PR TITLE
New node to import known poses for various file formats

### PR DESCRIPTION
**Pull request by fabiencastan**
Based on https://github.com/alicevision/AliceVision/pull/1078

### CODE REVIEW

File: meshroom/nodes/aliceVision/ImportKnownPoses.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+from meshroom.core import desc
+
+
+class ImportKnownPoses(desc.CommandLineNode):
+    commandLine = 'aliceVision_importKnownPoses {allParams}'
+    size = desc.DynamicNodeSize('sfmData')
+
+    documentation = '''
+    Import konwn poses from varuois file format like xmp or json.
```
Comment (natowi) [inline meshroom/nodes/aliceVision/ImportKnownPoses.py:11]: Typo in "known"

**Comment by flolu** [comment]
@fabiencastan does this node make it possible to do texturing with just a known mesh, known poses and input images?

**Comment by Ben-Mack** [comment]
@flolu Did you success doing texturing with ImportKnownPoses node?
@fabiencastan I've the same question, can you confirm this ImportKnownPoses node can do Texturing given known mesh, known poses (without running other steps)?

**Comment by flolu** [comment]
@Ben-Mack unfortunately, I wasn't able to do it.

**Pull request merged**